docs/development/doc_guide.rst

```diff
@@ -60,17 +343,4 @@ Narrative Documentation
 * Each subpackage must have narrative documentation describing its
   use.
 
-Substitutions
-=============
-
-The `common_links.rst
-<https://github.com/PlasmaPy/PlasmaPy/blob/master/docs/common_links.rst>`_
-file defines reStructuredText substitutions that can be used in
-PlasmaPy's documentation.
-
-.. code-block:: RST
-
-    Instead of writing `~plasmapy.particles.particle_class.Particle`,
-    we can write |Particle|.
-
-Substitutions should be defined when they are used in multiple files.
+.. replace:: _`Read the Docs`: https://readthedocs.org/
```

Comment:
```suggestion
.. _`Read the Docs`: https://readthedocs.org/
```

This is what is causing the current failed check.

Reply:
Good catch — thanks!